### DISCUSSION TITLE
Imagenet1k is too big!

### DISCUSSION THREAD

**Discussion by mrFocusXin** [open-mmlab/mmdetection#9826] [Q&A] [answered]
**As title, The Imagenet1k is too big to me, my machine can't work with Imagenet1k .  But i still want to run this program, how should i do ? I edit th batch size smaller,but it still be killed because excessive memory usage. Does anyone have any ideas or experience? Your reply will be very helpful to me!**

![image](https://user-images.githubusercontent.com/41821856/220852627-92d77317-6bce-4fed-a6e6-791ac97e09e5.png)

**Comment by ZwwWayne** [comment, marked as answer]
You should create an issue in https://github.com/open-mmlab/mmselfsup/discussions

**Reply by mrFocusXin** [reply]
OK, thank you.
